### ISSUE TITLE
Failed to load the hey-pane package

### ISSUE THREAD

**Issue by viddo** [timomeh/hey-pane#5]
Started to appear after updating to latest Atom Beta version (was released today I believe).

1. Open Atom Beta (https://atom.io/beta)

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [hey-pane](https://github.com/timomeh/hey-pane) package 0.2.1

### Stack Trace

Failed to load the hey-pane package

```
At /Users/nicklas/.atom-beta/packages/hey-pane/lib/hey-pane.js: package is a reserved word in strict mode (134:43)

SyntaxError: /Users/nicklas/.atom-beta/packages/hey-pane/lib/hey-pane.js: package is a reserved word in strict mode (134:43)
  132 | 
  133 |   checkIncompatiblePackages(package) {
> 134 |     if (this.incompatiblePackages.includes(package.name)) {
      |                                            ^
  135 |       atom.notifications.addError('Incompatible Package Detected',
  136 |         {
  137 |           dismissable: true,
    at /app.asar/node_modules/babylon/lib/index.js:4373:13)
    at /app.asar/node_modules/babylon/lib/index.js:4270:10)
    at /app.asar/node_modules/babylon/lib/index.js:4243:10)
    at /app.asar/node_modules/babylon/lib/index.js:3581:21)
    at Parser.parseExprAtom (/app.asar/node_modules/babylon/lib/index.js:7016:22)
    at /app.asar/node_modules/babylon/lib/index.js:3427:19)
    at /app.asar/node_modules/babylon/lib/index.js:3407:19)
    at /app.asar/node_modules/babylon/lib/index.js:3337:19)
    at /app.asar/node_modules/babylon/lib/index.js:3314:19)
    at /app.asar/node_modules/babylon/lib/index.js:3277:19)
    at Parser.parseMaybeAssign (/app.asar/node_modules/babylon/lib/index.js:6242:20)
    at /app.asar/node_modules/babylon/lib/index.js:4231:16)
    at Parser.parseExprListItem (/app.asar/node_modules/babylon/lib/index.js:5913:29)
    at /app.asar/node_modules/babylon/lib/index.js:3506:20)
    at /app.asar/node_modules/babylon/lib/index.js:3466:31)
    at /app.asar/node_modules/babylon/lib/index.js:3437:15)
    at /app.asar/node_modules/babylon/lib/index.js:3407:19)
    at /app.asar/node_modules/babylon/lib/index.js:3337:19)
    at /app.asar/node_modules/babylon/lib/index.js:3314:19)
    at /app.asar/node_modules/babylon/lib/index.js:3277:19)
    at Parser.parseMaybeAssign (/app.asar/node_modules/babylon/lib/index.js:6242:20)
    at /app.asar/node_modules/babylon/lib/index.js:3239:19)
    at /app.asar/node_modules/babylon/lib/index.js:3722:18)
    at /app.asar/node_modules/babylon/lib/index.js:2013:20)
    at /app.asar/node_modules/babylon/lib/index.js:1801:19)
    at Parser.parseStatement (/app.asar/node_modules/babylon/lib/index.js:5712:22)
    at /app.asar/node_modules/babylon/lib/index.js:2223:21)
    at /app.asar/node_modules/babylon/lib/index.js:2202:8)
    at /app.asar/node_modules/babylon/lib/index.js:4154:22)
    at Parser.parseFunctionBody (/app.asar/node_modules/babylon/lib/index.js:5699:20)
```

### Commands

```
```

### Non-Core Packages

```
autocomplete-paths 1.0.5 
date 1.2.2 
file-icons 2.0.17 
gfm-lists 1.0.0 
hey-pane 0.2.1 
imdone-atom 2.1.22 
jumpy 3.1.3 
minimap 4.26.8 
minimap-find-and-replace 4.5.2 
minimap-highlight-selected 4.5.0 
quick-file-actions 0.1.1 
recent-files-fuzzy-finder 0.3.0 
textual-velocity 0.12.4 
toggle-markdown-task 0.6.0 
vim-mode-plus 0.83.0 
wikilink 0.14.0 
zentabs 0.8.8 
```

**Comment by timomeh** [comment]
Thanks! I'll update it asap.

**Comment by timomeh** [comment]
I just published version v0.2.2 with a fix for this.

Thank you very much for reporting this issue!

**Comment by viddo** [comment]
That was fast! Thanks :)
